Src/Particle/AMReX_ParticleTile.H

```diff
@@ -746,11 +734,7 @@ struct ParticleTile
     SoA&       GetStructOfArrays ()       { return m_soa_tile; }
     const SoA& GetStructOfArrays () const { return m_soa_tile; }
 
-    template <typename T = ParticleType, typename std::enable_if<!T::is_soa_particle, int>::type = 0>
-    bool empty () const { return m_aos_tile.empty(); }
-
-    template <typename T = ParticleType, typename std::enable_if<T::is_soa_particle, int>::type = 0>
-    bool empty () const { return m_soa_tile.empty(); }
+    bool empty () const { return size() == 0; }
```

Comment:
The development version for `empty` also works, it was fixed after I made the original PR

Reply:
True, but removing the SFINAE will improve compile time :)